builtin/providers/nsone/provider.go

```diff
@@ -32,9 +35,10 @@ func Provider() terraform.ResourceProvider {
 }
 
 func nsoneConfigure(d *schema.ResourceData) (interface{}, error) {
-	n := nsone.New(d.Get("apikey").(string))
-	n.Debug()
-	n.RateLimitStrategySleep()
+	httpClient := &http.Client{}
+	n := nsone.NewClient(httpClient, nsone.SetAPIKey(d.Get("apikey").(string)))
+	// FIXME: n.Debug()
+	// n.RateLimitStrategySleep()
```

Comment:
I got a divide by zero panic that I haven't had a change to investigate. I absolutely want to re-enable it.